cardano-api/cardano-api.cabal

```diff
@@ -61,6 +61,7 @@ library
                         Cardano.Api.Hash
```

Comment:
I would suggest that the commit subject line should be "Merge query tip queries" and the rest could go in the message body.

Reply:
Pushed the new commit message.